packages/main/src/themes/CalendarHeader.css

```diff
@@ -131,3 +131,23 @@
 .ui5-calheader-middlebtn:focus:active::after {
 	border-color: var(--sapContent_ContrastFocusColor);
 }
+
+/* Todo: Remove with introduction of YearRangePicker */
+.ui5-calheader-middlebtn.ui5-calheader-yearrange-btn-disabled:hover,
+.ui5-calheader-middlebtn.ui5-calheader-yearrange-btn-disabled:active,
+.ui5-calheader-middlebtn.ui5-calheader-yearrange-btn-disabled:focus,
+.ui5-calheader-middlebtn.ui5-calheader-yearrange-btn-disabled:focus:active,
+.ui5-calheader-middlebtn.ui5-calheader-yearrange-btn-disabled {
+	cursor: default;
```

Comment:
Testing around I see that, leaving only the `pointer-events` achieves the same, therefore, I don't think we need the rest of the properties.

I see that those selectors will be removed when the Year Range Picker is introduced, so I am good with the change either way.